src/test/java/net/masterthought/cucumber/generators/integrations/helpers/EmbeddingAssertion.java

```diff
@@ -19,7 +19,8 @@ public void hasImageContent(Embedding embedding) {
     }
 
     public void hasTextContent(String content) {
-        assertThat(getBox().oneBySelector("pre", WebAssertion.class).html()).isEqualTo(getDecodedData(content));
+        assertThat(getBox().oneBySelector("pre", WebAssertion.class).text())
+                .isEqualTo(getDecodedData(content));
     }
```

Comment:
Replaced .html with .text so the decoded data doesn't need to be escaped to match up.